### PR TITLE
Issue-63 new Dockerfile

### PR DESCRIPTION
**Pull request by caa84**
Docker file runs the server

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -37,6 +36,6 @@ ENV APP_ENV test
 # Ensure all python requirements are met
 RUN pip install -r /opt/app/requirements.txt
 
-CMD ["/bin/bash"]
-
 EXPOSE 8000
+
+CMD ["python","opt/app/bin/manage.py","runserver","0.0.0.0:8000"]
```
Comment (bruth) [inline Dockerfile]: Add the leading slash to the path so it is absolute. Add a space between elements in the array.

**Comment by bruth** [comment]
Also, please squash the commits and ensure the commit message reflects the change that was made (Default command now runs the server). @tjrivera can help you with that.

**Comment by tjrivera** [comment]
There's also a few steps that you'll need the application to do before you run the `runserver` command. You'll want to execute the `make setup` in the project's root directory to setup the local_settings.py file. You'll also want to run the `syncdb` django management command. 

These steps could be incorporated in initialization script that the Dockerfile would execute at runtime (instead of the runserver command)

**Comment by tjrivera** [comment]
You've also go a fair amount of python packages in your requirements.txt file that aren't being installed in the Dockerfile. If you keep these two files in sync, your builds will be much faster because they're cached.

**Comment by tjrivera** [comment]
You might consider using an initialization script that lives at `scripts/run.sh` that handles your application's initialization. 

Something like:

```bash
#!/bin/bash

APP_DIR = /opt/app/

cd $APP_DIR

make setup
make collect

python /opt/app/bin/manage.py syncdb --noinput

python /opt/app/bin/manage.py runserver 0.0.0.0:8000
```

Your Dockerfile's `CMD` directive would then become `["/opt/app/scripts/run.sh"].

Make sure you make the `run.sh` file executable on your local machine. Those permissions flow through to the docker container. You would do that by running `sudo chmod +x scripts/run.sh` on your local machine. You should only have to do this once.

**Pull request merged**